src/wrapper/signature.rs

```diff
@@ -133,58 +136,70 @@ where
         .or(void))
     .map(JavaType::Primitive)
     .parse_stream(input)
+    .into()
 }
 
-fn parse_array<S: Stream<Item = char>>(input: &mut S) -> ParseResult<JavaType, S>
+fn parse_array<S: Stream<Token = char>>(input: &mut S) -> StdParseResult<JavaType, S>
 where
     S::Error: ParseError<char, S::Range, S::Position>,
 {
-    let marker = token('[');
-    (marker, parser(parse_type))
+    let marker = combine::token('[');
+    (marker, combine::parser(parse_type))
         .map(|(_, ty)| JavaType::Array(Box::new(ty)))
         .parse_stream(input)
+        .into()
 }
 
-fn parse_object<S: Stream<Item = char>>(input: &mut S) -> ParseResult<JavaType, S>
+fn parse_object<S: Stream<Token = char>>(input: &mut S) -> StdParseResult<JavaType, S>
 where
     S::Error: ParseError<char, S::Range, S::Position>,
 {
-    let marker = token('L');
-    let end = token(';');
-    let obj = between(marker, end, many1(satisfy(|c| c != ';')));
+    let marker = combine::token('L');
+    let end = combine::token(';');
+    let obj = combine::between(marker, end, combine::many1(combine::satisfy(|c| c != ';')));
```

Comment:
Shall we leave qualified references, or keep imports?

Reply:
I'm not sure myself. Using wildcard imports is considered bad practice, though I don't think that importing all these functions manually would look better. Additionally, style guides recommend to only import parent module for functions instead of importing them directly.